integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -160,6 +162,88 @@ def _check_matched_content_updated_at(
                 items_to_update[matched_item.get('content_key')] = item
         return items_to_update
 
+    def _check_matched_content_to_create(
```

Comment:
before this refactoring, `ContentMetadataItemTransmission` were only created or undeleted _after_ they were successfully transmitted to the remote system. this refactor pre-creates them instead so we have an object to capture error messages and status on. this method as well as the semantic of pre-creating these records adds more consistency with how data flows.